platforms-serverless-vercel/vercel-cli-cache-repro/prisma/schema.prisma

```diff
@@ -0,0 +1,16 @@
+// Like, postgresql://user:password@localhost:5432/database/schema
+datasource db {
+  provider = "postgresql"
+  url      = env("VERCEL_API_CACHE_REPRO_PG_URL")
+}
+
+generator client {
+  provider = "prisma-client-js"
+}
+
+model User {
+  id    String @id @default(cuid())
+  email String @unique
+  // Here we remove it
+  // name  String
+}
```

Comment:
Is this the correct schema file to have as default `schema.prisma`?

Reply:
Possible `schema.prisma` should be on .gitignore if I understood the script below correctly.

Reply:
It doesn't really matter because it's always overwritten